src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -457,23 +453,20 @@ def _record_compile_classpath(self, classpath, targets, outdir):
       with open(path, 'w') as f:
         f.write(text.encode('utf-8'))
 
-  def _compile_vts(self, vts, target, sources, analysis_file, upstream_analysis, classpath, outdir,
-                   log_dir, zinc_args_file, progress_message, settings, fatal_warnings,
+  def _compile_vts(self, vts, ctx, upstream_analysis, classpath, progress_message, settings, fatal_warnings,
                    zinc_file_manager, counter):
     """Compiles sources for the given vts into the given output dir.
 
-    vts - versioned target set
-    sources - sources for this target set
-    analysis_file - the analysis file to manipulate
-    classpath - a list of classpath entries
-    outdir - the output dir to send classes to
+    :param vts: VersionedTargetSet with one entry for the target.
+    :param ctx: - A CompileContext instance for the target.
```

Comment:
Nit: Delete the leading dash.

Reply:
I'll fix this and expand the tests in a followup.